src/test/java/com/iexec/sms/Web3jUtils.java

```diff
@@ -0,0 +1,36 @@
+/*
+ * Copyright 2022 IEXEC BLOCKCHAIN TECH
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.iexec.sms;
+
+import org.web3j.crypto.Credentials;
+import org.web3j.crypto.ECKeyPair;
+import org.web3j.crypto.Keys;
+
+public class Web3jUtils {
+
+    private Web3jUtils() {}
+
+    public static String getEthereumAddress() {
```

Comment:
Shouldn't this return an `Optional`?

Reply:
@mcornaton, what would bring an optional here ?
My idea was to avoid the need for boilerplate code in tests.

We may return a `null` value instead of `""` to provoke a NPE and fail test faster.
With an official JVM, the exceptions thrown by `createEcKeyPair` should never be thrown.